CMakeLists.txt

```diff
@@ -2,10 +2,12 @@ cmake_minimum_required(VERSION 3.14)
 include(GNUInstallDirs)
 project(hiredis-cluster)
 
+# Options
 option(DOWNLOAD_HIREDIS  "Download the dependency hiredis from GitHub" ON)
 option(ENABLE_SSL        "Enable SSL/TLS support" OFF)
 option(DISABLE_TESTS     "Disable compilation of test" OFF)
 option(ENABLE_IPV6_TESTS "Enable IPv6 tests requiring special prerequisites" OFF)
+# USE_SANITIZER - Build using sanitizer "address", "thread", "undefined", "leak", ..
```

Comment:
Remove this?

Reply:
was thinking that this could be describing all options, incl non-on/off...but then again, its in the readme, and can be found anyway so removed it